frontend/src/libs/components/header/helpers/get-burger-menu-items.helper.ts

```diff
@@ -0,0 +1,23 @@
+import { type BurgerMenuItem } from '~/libs/types/types.js';
+import { UserGroupKey } from '~/packages/users/libs/enums/enums.js';
+
+import { BURGER_MENU_ITEMS } from '../burger-menu-items/burger-menu-items.js';
+
+const getBurgerMenuItems = (group: string): BurgerMenuItem[] | null => {
+  switch (group) {
+    case UserGroupKey.BUSINESS: {
+      return BURGER_MENU_ITEMS.BusinessMenu;
+    }
+    case UserGroupKey.CUSTOMER: {
+      return BURGER_MENU_ITEMS.CustomerMenu;
+    }
```

Comment:
can we return customer menu as the default option? to avoid returning null value 
because there is no need in null value now, as BurgerMenu is rendered only for authenticated users who have menu items